Source/Core/DolphinQt2/GameList/GameList.cpp

```diff
@@ -105,18 +115,54 @@ void GameList::MakeListView()
 
 void GameList::ShowContextMenu(const QPoint&)
 {
+  const auto game = GetSelectedGame();
+  if (game.isEmpty())
+    return;
+
   QMenu* menu = new QMenu(this);
-  DiscIO::Platform platform = GameFile(GetSelectedGame()).GetPlatformID();
+  DiscIO::Platform platform = GameFile(game).GetPlatformID();
+  menu->addAction(tr("Properties"), this, SLOT(OpenProperties()),
+                  QKeySequence(tr("P", "Properties")));
+  menu->addAction(tr("Wiki"), this, SLOT(OpenWiki()), QKeySequence(tr("W", "Wiki")));
+  menu->addSeparator();
+
   if (platform == DiscIO::Platform::GAMECUBE_DISC || platform == DiscIO::Platform::WII_DISC)
   {
-    menu->addAction(tr("Properties"), this, SLOT(OpenProperties()));
-    menu->addAction(tr("Open Wiki Page"), this, SLOT(OpenWiki()));
-    menu->addAction(tr("Set as Default ISO"), this, SLOT(SetDefaultISO()));
+    menu->addAction(tr("Default ISO"), this, SLOT(SetDefaultISO()),
+                    QKeySequence(tr("D", "Set as Default ISO")));
+    menu->addAction(GameFile(game).GetBlobType() == DiscIO::BlobType::GCZ ? tr("Decompress ISO") :
+                                                                            tr("Compress ISO"),
```

Comment:
"Compress ISO" is only supposed to be available when the BlobType is Plain. The compression code doesn't work right for any other format.